### PR TITLE
Clarify performance overlay in performance issue template

### PR DESCRIPTION
**Pull request by goderbauer**

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/5_performance_speed.md [hunk 1]
```diff
@@ -46,6 +45,7 @@ assignees: ''
      so we know which functions might be causing it. See "How to Collect
      and Read Timeline Traces" on this blog post:
        https://medium.com/flutter/profiling-flutter-applications-using-the-timeline-a1a434964af3#a499
+     Make sure the performance overlay is turned OFF when recording the trace.
```
Comment (cbracken) [inline .github/ISSUE_TEMPLATE/5_performance_speed.md:48]: Consider adding a brief explanation of how to do that -- i.e. press 'P' on the command line or the equivalent IDE action.

**Review by cbracken — APPROVED** [review]

File: .github/ISSUE_TEMPLATE/5_performance_speed.md (hunk 1 above)
Comment (goderbauer) [reply .github/ISSUE_TEMPLATE/5_performance_speed.md]: Added a note.

File: .github/ISSUE_TEMPLATE/5_performance_speed.md [hunk 2]
```diff
@@ -46,6 +45,8 @@ assignees: ''
      so we know which functions might be causing it. See "How to Collect
      and Read Timeline Traces" on this blog post:
        https://medium.com/flutter/profiling-flutter-applications-using-the-timeline-a1a434964af3#a499
+     Make sure the performance overlay is turned OFF when recording the
+     trace (Pressing ‘P’ on the command line toggles the overlay).
```
Comment (dnfield) [inline .github/ISSUE_TEMPLATE/5_performance_speed.md:49]: nit: We could mention that this is a useful tool for getting a general idea, but that the tool itself has more overhead than just tracing and tracing should be preferred.

**Review by dnfield — APPROVED** [review]
LGTM with nit

File: .github/ISSUE_TEMPLATE/5_performance_speed.md [hunk 3]
```diff
@@ -46,6 +45,9 @@ assignees: ''
      so we know which functions might be causing it. See "How to Collect
      and Read Timeline Traces" on this blog post:
        https://medium.com/flutter/profiling-flutter-applications-using-the-timeline-a1a434964af3#a499
+     Make sure the performance overlay is turned OFF when recording the
+     trace as that may affects the performance of the profile run.
```
Comment (MrBirb) [inline .github/ISSUE_TEMPLATE/5_performance_speed.md:49]: nit: affect*

**Pull request merged**